CodeModel/src/main/java/org/openzen/zenscript/codemodel/FunctionHeader.java

```diff
@@ -469,4 +470,8 @@ public int hashCode() {
 		result = 31 * result + (hasUnknowns ? 1 : 0);
 		return result;
 	}
+
+	public boolean hasInvalidTypes() {
+		return Stream.of(parameters).anyMatch(p -> p.type.isInvalid());
```

Comment:
💭 Thought: While writing this I found out we have `BasicTypeID.INVALID` and `InvalidTypeID`.
Do we need to change some calls that only compare for `== BasicTypeID.INVALID` to check for `isInvalid()` instead?